src/main/java/mate/academy/spring/config/SecurityConfig.java

```diff
@@ -28,10 +28,12 @@ public void configure(AuthenticationManagerBuilder auth) throws Exception {
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .authorizeRequests()
+                .antMatchers(HttpMethod.POST,"/register/**").hasAnyRole()
                 .antMatchers(HttpMethod.POST,"/movies/**").hasRole("ADMIN")
-                .antMatchers(HttpMethod.GET,"/movies/**").permitAll()
+                .antMatchers(HttpMethod.GET,"/movies/**").hasAnyRole()
```

Comment:
and here even better to use `.hasAnyAuthority("ADMIN", "USER")`
replace other `hasAnyRole`

Reply:
I added `.hasAnyAuthority("ADMIN", "USER")` instead, but I get 403 Forbidden for user and admin. It doesn't work without `ROLE_` prefix